istio/README.md

```diff
@@ -24,18 +24,14 @@ If you want to monitor the Envoy proxies in Istio, configure the [Envoy integrat
 
 ### Configuration
 
-Edit the `istio.d/conf.yaml` file (in the `conf.d/` folder at the root of your [Agent's configuration directory][7]) to connect to Istio. See the [sample istio.d/conf.yaml][8] for all available configuration options.
```

Comment:
I think we should leave this statement in, since all checks use this wording

Reply:
Added back in 👍

Reply:
Thanks!